src/vendor_specific/gce/create-lb.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+source ./prepare_env.sh false
+
+HEALTH_CHECK_NAME=$TEMPLATE_NAME"-health-check"
+BACKEND_SERVICE_NAME=$TEMPLATE_NAME"-lb"
+FORWARDING_RULE_NAME=$TEMPLATE_NAME"-forward-rule"
+
+if [ -z "$LB_NETWORK" ]
+then
+    LB_NETWORK=$NETWORK
+fi
+
+if [ -z "$LB_SUBNET" ]
+then
+    LB_SUBNET=$SUBNET
+fu
```

Comment:
typo `fi`

Reply:
Just see this while drinking water... almost choke 😂